app/views/stores/index.html.erb

```diff
@@ -1,15 +1,20 @@
 <h1>Nearby Stores</h1>
 <% @stores.each do |store| %>
-<iframe
-  width="400"
-  height="400"
-  frameborder="0" style="border:0"
-  src="https://www.google.com/maps/embed/v1/place?key=<%= store.google_embed_key  %>&q=place_id:<%= store.place_id  %>">
-</iframe><br>
-  <%= store.name %><br>
-  <%= store.address %><br>
-  <%= store.open_now %><br>
-  <%= store.google_rating %><br>
-  <%= store.operating_hours %><br>
-  <br><br>
+  <div id="store-map">
+    <iframe
+      width="400"
+      height="400"
+      frameborder="0" style="border:0"
+      src="https://www.google.com/maps/embed/v1/place?key=<%= store.google_embed_key  %>&q=place_id:<%= store.place_id  %>">
```

Comment:
@s-espinosa, any other way to make this work without the iframe call? (this is an interactive google map btw)